### PR TITLE
chore/cleanup

### PR DESCRIPTION
**Pull request by bwalsh**
This PR:
* has no new features, purely cleanup
* removes dead code and unused commands
* cleans up command vocabulary
* moves meta skeleton generation test to tests/integration
* preparation for #34

### CODE REVIEW

File: gen3_util/access/requestor.py [hunk 1]
```diff
@@ -198,8 +179,8 @@ def add_policies(config: Config, project_id: str) -> LogAccess:
         requests.append(cp(request=policy, config=config).request)
         request_ids.append(requests[-1]['request_id'])
 
-    commands = [f"gen3_util access update {request_id} SIGNED" for request_id in request_ids]
-    msg = f"Approve these requests to assign default policies to {project_id}"
+    commands = ["gen3_util access sign"]
+    msg = f"Approve these requests to create assign default policies to {project_id}"
```
Comment (lbeckman314) [inline gen3_util/access/requestor.py]: Minor possible typo, should this line be:

```py
msg = f"Approve these requests to create default policies for {project_id}"
```

**Review by lbeckman314 — APPROVED** [review]
## Overview

Looks good!

All tests passing except for [test_incremental_workflow](https://github.com/ACED-IDP/gen3_util/blob/chore/cleanup-unused/tests/integration/test_meta.py#L192) which is failing due to a `403` error when Fence tries to delete the record from Indexd.

This might be due to incorrect Indexd credentials in the local **fence-config.yaml**, so not due to anything in gen3_util.

## Pytest in VSCode

**.env file** (required to use the 'local' profile in Pytest):
```env
GEN3_UTIL_PROFILE=local
```

**fence-config.yaml:**
```
      aced-ohsu-development:
        cred: OHSU-USER
        endpoint_url: https://aced-storage.ohsu.edu/
        programs:
        - test   <--- Required for tests
```

**values.yaml:**
```yaml
fence:
  image:
    repository: quay.io/ohsu-comp-bio/fence
    pullPolicy: Always
    tag: "feat_s3-delete-files"   <--- Required to delete files from non-AWS buckets
```

**Pytest results:**
Screenshot 2023-12-26 at 1 08 55 PM

## Fence/Indexd Error

```log
[2023-12-26 20:59:26,651][     fence][   INFO] deleted file for prefix 5636257a-7f99-5bd3-9382-78c54d0f535e/tests/fixtures/dir_to_study/sub-dir/file-5 in bucket aced-ohsu-development
[2023-12-26 20:59:26,745][fence.blueprints.data.indexd][  ERROR] Unable to delete indexd record '5636257a-7f99-5bd3-9382-78c54d0f535e': <Response [403]>
10.42.0.189 - - [26/Dec/2023:20:59:26 +0000] "DELETE /data/5636257a-7f99-5bd3-9382-78c54d0f535e HTTP/1.0" 500 46 "-" "python-requests/2.31.0" "10.42.0.1"
```

## fence-config.yaml

```yaml
    # this is the username which fence uses to make authenticated requests to indexd
    INDEXD_USERNAME: 'fence'
    # this is the password which fence uses to make authenticated requests to indexd
    INDEXD_PASSWORD: ''   <--- Should this be set to a value in values.yaml?
```

## Fence/Indexd Code

https://github.com/ohsu-comp-bio/fence-multipart/blob/feat/s3-delete-files/fence/blueprints/data/indexd.py#L758-L763

## Update

All tests passing and able to successfully delete files from buckets as well as from Indexd.

**Changes:**
1. Update Fence PR to use the correct credentials for each bucket. [Previously](https://github.com/uc-cdis/fence/pull/1127/files#diff-0bb0aa162c44af6b944062be516bba755070faed736c61a535add829eb037e65L391-L392) it would only pass in the first AWS_CREDENTIALS block from fence-config.yaml. [Now](https://github.com/uc-cdis/fence/pull/1127/files#diff-6f95d12e14bc9bc31c18dc66282578796bfeacb49b76815bf8c2ac11230aee6aR35-R36) the S3 connection manager is passed all credentials with the correct credentials being assigned to each bucket.
2. Had to manually delete the existing service accounts in the indexd database (auth_record) and restart the indexd-userdb job by redeploying. This recreated the service accounts in the database with the [correct passwords](https://github.com/ACED-IDP/gen3-helm/blob/develop/helm/indexd/templates/pre-install.yaml#L66-L71). Not sure why the old passwords were still persisting in the database?

Screenshot 2023-12-26 at 5 11 10 PM

## Additional Resources:
- The [template](https://github.com/ACED-IDP/gen3-helm/blob/develop/helm/fence/values.yaml#L285-L291) for Fence connecting to Indexd.
- The [code that sends the DELETE request ](https://github.com/ohsu-comp-bio/fence-multipart/blob/feat/s3-delete-files/fence/blueprints/data/indexd.py#L758)from Fence to Indexd.
- The [Auth code in Indexd](https://github.com/uc-cdis/indexd/blob/575d497a275e1d285bebc2eec62476ef99db5cd5/indexd/auth/drivers/alchemy.py#L105-L109) that was outputing the 403 error before manually recreating the auth_record with the indexd-userdb job.

File: gen3_util/access/requestor.py (hunk 1 above)
Comment (bwalsh) [reply gen3_util/access/requestor.py]: @lbeckman314  Thanks for the detailed review.  I will make the suggested `msg` change.    Should we add the "fence over view" somewhere in the docs?  Or, is it a passing dependency that will be resolved when merged?

Comment (lbeckman314) [reply gen3_util/access/requestor.py]: It is a passing dependency that will be resolved when merged, no changes required anywhere else.

**Pull request merged**